.github/workflows/java8_unit_tests.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         timeout-minutes: 60
       - name: Generate coverage report
         id: cov0
+        if: always()
```

Comment:
if the tests fail, do we really want to generate the coverage report?

Reply:
Yes. Because sending the failed report is the main way codecov knows that the ci build failed.